README.md

```diff
@@ -14,9 +14,15 @@ Before you begin, make sure the latest version of the [Notation CLI has been ins
 1. Navigate to the [Releases](https://github.com/Azure/notation-azure-kv/releases) page and choose a release of `notation-azure-kv`.
 2. Download, verify, and then install the specified version of the plugin.
 
+   **Automatically install**:
+
+   For Notation >= v1.1.0, please use [notation plugin install](https://github.com/notaryproject/notation/blob/v1.1.0/specs/commandline/plugin.md#notation-plugin-install) command to automatically install azure-kv plugin.
```

Comment:
What's the command for `notation plugin install` for installing `v1.0.2`?

Reply:
We don't have the checksum before relase v1.0.2. We can add commands to the README after release v1.0.2.

Reply:
Added the commands.